docs/cloud/aws/getting-started/non-cu-boulder/aws-console-access-non-cu-boulder.md

```diff
@@ -1,13 +1,13 @@
 # Accessing the AWS Management Console with CILogon (Non-CU Boulder)
 
 ## Prerequisites
-This guide is only relevant if you are not a member of CU Boulder and require access to manage AWS resources owned by CU Boulder. If you are a member of CU Boulder, please see <a href="./aws-console-access.md" target="_blank">this guide</a> on signing in using your IdentiKey.
+This guide is only relevant if you are not a member of CU Boulder and require access to manage AWS resources owned by CU Boulder. If you are a member of CU Boulder, please see [this guide](../aws-console-access.html) on signing in using your IdentiKey.
```

Comment:
For internal references, you should not use `.html`, instead, `.md` should be used.

Reply:
ccb0f5c